students/JonathanMauk/lesson05/mailroom03.py

```diff
@@ -0,0 +1,148 @@
+import os
+import sys
+
+donor_names = ["John Smith", "Jane Doe", "Alan Smithee", "Tom D.A. Harry", "Joe Shmoe"]
+donation_amounts = [[18774.48, 8264.47, 7558.71], [281918.99, 8242.13], [181.97, 955.16], [67.10, 500.98], [200.01]]
+
+donor_db = {name: donation for name, donation in zip(donor_names, donation_amounts)}
+# donor_db = {"John Smith": [18774.48, 8264.47, 7558.71], "Jane Doe": [281918.99, 8242.13],
```

Comment:
This works for getting in practice with comprehensions, but in reality would probably be more appropriate to define as dict literal.